packages/resolver/solidity/Console.sol

```diff
@@ -0,0 +1,65 @@
+pragma solidity >=0.4.21 <0.7.0;
```

Comment:
Do we need to update this to support `^0.7.0` now?

Reply:
Yeah I suppose this should have a `<0.8.0`, right?

Reply:
yep and it might have some other changes required

Reply:
updated  the pragma